scene/main/viewport.h

```diff
@@ -751,6 +751,7 @@ class SubViewport : public Viewport {
 
 public:
 	void set_size(const Size2i &p_size);
+	void internal_set_size(const Size2i &p_size, const bool p_force = false);
```

Comment:
We don't use `const` for bool/float/int.

Reply:
Thanks, I corrected that.